common/src/test/java/gov/cms/ab2d/common/EntityUtils.java

```diff
@@ -0,0 +1,72 @@
+package gov.cms.ab2d.common;
+
+import gov.cms.ab2d.common.model.*;
+import gov.cms.ab2d.common.repository.*;
+
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.util.Random;
+
+public class EntityUtils {
```

Comment:
I don't like the concept of this class.  Put a static create method on each domain class and then where used, it's simply a repo.save(Domain.create(...)) call.

Reply:
These should be in the service classes for each entity it's related to, not the domain class. They use the repository, which is a bean, the repo is already in the service class.

Reply:
Actually I misread this, it's for testing purposes. I think it's fine to have it stay here, although I think there's already a bean that does some data creation

Reply:
DataSetup.java is the bean if you can use it. That way you don't have to pass the repo into every method.

Reply:
I'll move everything to DataSetup instead.